.github/CODEOWNERS

```diff
@@ -437,13 +437,13 @@
 /test/kitchen/test/integration/win-user/ @DataDog/windows-agent
 /test/fakeintake/                       @DataDog/agent-e2e-testing
 /test/new-e2e/                          @DataDog/agent-e2e-testing
-/test/new-e2e/agent-subcommands         @DataDog/agent-shared-components
 /test/new-e2e/language-detection        @DataDog/processes
-/test/new-e2e/ndm                       @DataDog/network-device-monitoring
-/test/new-e2e/scenarios/ndm             @DataDog/network-device-monitoring
 /test/new-e2e/system-probe              @DataDog/ebpf-platform
 /test/new-e2e/scenarios/system-probe    @DataDog/ebpf-platform
+/test/new-e2e/tests/agent-subcommands   @DataDog/agent-shared-components
```

Comment:
do we really need to repeat `tests` in the path?

Reply:
I agree it is not ideal, could not come up with another name
Did not want to call it `teams` as we rather have a mix of features and team folders. Do you have suggestions ?